site/utils/index.ts

```diff
@@ -49,9 +49,36 @@ export const setZodDefaultError: (errorMessage: string) => { errorMap: ZodErrorM
 
 const dateRegex = /^(0?[1-9]|[12][0-9]|3[01])[\/](0?[1-9]|1[012])[\/]\d{4}$/;
 const timeRegex = /^([0-1][0-9]|2[0-3])[0-5][0-9]$/;
+const hoursAndMinutesRegex = /^([0-9]{0,4})[0-5][0-9]$/;
 
 export const zodDate = (defaultError?: string) =>
     z.string(defaultError ? setZodDefaultError(defaultError) : {}).regex(dateRegex);
 
 export const zodTime = (defaultError?: string) =>
     z.string(defaultError ? setZodDefaultError(defaultError) : {}).regex(timeRegex);
+
+export const zodTimeInHoursAndMinutes = (defaultError?: string) =>
```

Comment:
There are some functions here with regex's that we pass in, could we add a unit test for each? If only to show what they actually do, because I cant tell by reading the zod code.

Reply:
Yeah that would be useful - it means if someone else comes along who hasnt touched this area of code, they can use the functions and know what the output will be

Reply:
done